app/frontend/src/components/RedditStylePostCard/__tests__/MediaPreview.test.tsx

```
@@ -0,0 +1,429 @@
+import React from 'react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import MediaPreview from '../MediaPreview';
+import { thumbnailService } from '@/services/thumbnailService';
+
+// Mock framer-motion
+jest.mock('framer-motion', () => ({
+  motion: {
+    div: ({ children, ...props }: any) => <div {...props}>{children}</div>,
+    img: ({ children, ...props }: any) => <img {...props}>{children}</img>,
```

Biome (2.1.2)

[error] img is a void element tag and must not have children.

Unsafe fix: Remove the children.


(lint/correctness/noVoidElementsWithChildren)